### PR TITLE
[#368] Search thumbnail draft

### LINKED ISSUE #368: Include subtitle in software listing page
I believe that the software listing page should also contain a subtitle.
Right now the title is not self explicative enough and it does not help the reader.

### PR DESCRIPTION
**Pull request by sebbalex**
Fixes #368

### CODE REVIEW

**Comment by libremente** [comment]
Hi @sebbalex could you please be a little more explicit in the PR text so that it's easier to understand what this is about? Thanks!

**Comment by sebbalex** [comment]
> Hi @sebbalex could you please be a little more explicit in the PR text so that it's easier to understand what this is about? Thanks!

This was original related to https://github.com/italia/developers.italia.it/issues/450. But later some of points listed were already fixed. Some others still to be fixed.
Furthermore this PR aims to add a brief description under every software in "Software listing" page and re-design that page to be more readable.

File: assets/scss/main.scss [hunk 1]
```diff
@@ -1172,8 +1172,8 @@ body {
                 left: 0;
                 content: ' ';
                 background-color: #E5F1FC;
-                z-index: -1;
                 width: 100%;
+                z-index: -1;
```
Comment (bfabio) [inline assets/scss/main.scss:1176]: Is he switch needed?

File: assets/js/esDevelopersItaliaQuery.js [hunk 2]
```diff
@@ -1010,9 +1018,17 @@ esDevelopersItaliaQuery.prototype.renderPost = function (post) {
     screenshot = post.logo;
   }
 
+  var shortDescription = '';
+  if(post.description) {
```
Comment (bfabio) [inline assets/js/esDevelopersItaliaQuery.js:1022]:
```suggestion
  if (post.description) {
```

File: assets/js/esDevelopersItaliaQuery.js [hunk 3]
```diff
@@ -985,10 +985,18 @@ esDevelopersItaliaQuery.prototype.renderSoftware = function (software) {
     localisedName = description.localisedName;
   }
 
+  var shortDescription = '';
+  if(description.shortDescription) {
```
Comment (bfabio) [inline assets/js/esDevelopersItaliaQuery.js:989]:
```suggestion
  if (description.shortDescription) {
```

**Review by bfabio — APPROVED** [review]
:+1:

File: assets/scss/main.scss (hunk 1 above)
Comment (sebbalex) [reply assets/scss/main.scss:1176]: maybe not, it is inherited from father anyway, but we can fix it in a dedicated issue/pr

Comment (libremente) [reply assets/scss/main.scss:1176]: @sebbalex Did you already create the new issue for this?

**Pull request merged**